pkg/trtl/mock/counterparty.go

```diff
@@ -0,0 +1,361 @@
+package mock
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/rotationalio/honu/replica"
+	"github.com/trisacrypto/directory/pkg/trtl/pb/v1"
+	"github.com/trisacrypto/directory/pkg/trtl/peers/v1"
+	"github.com/trisacrypto/directory/pkg/utils/bufconn"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+const (
+	bufSize     = 1024 * 1024
+	GetRPC      = "trtl.v1.Get"
+	PutRPC      = "trtl.v1.Put"
+	DeleteRPC   = "trtl.v1.Delete"
+	IterRPC     = "trtl.v1.Iter"
+	BatchRPC    = "trtl.v1.Batch"
+	CursorRPC   = "trtl.v1.Cursor"
+	SyncRPC     = "trtl.v1.Sync"
+	StatusRPC   = "trtl.v1.Status"
+	GetPeersRPC = "trtl.peers.v1.PeerManagement.GetPeers"
+	AddPeersRPC = "trtl.peers.v1.PeerManagement.AddPeers"
+	RmPeersRPC  = "trtl.peers.v1.PeerManagement.RmPeers"
+	GossipRPC   = "honu.replica.v1.Replication/Gossip"
```

Comment:
A bit of a mix and match of styles; but I'd propose that these be in the form `package.Service/RPC` to match the endpoints created by gRPC. E.g. `trtl.v1.Trtl/Get` or `trtl.peers.v1.PeerManagement/GetPeers`